v23.1/full-text-search.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Full-Text Search
+summary: Full-text searches using TSVECTOR and TSQUERY enable natural-language searches on documents with ranked results.
+toc: true
+docs_area: develop
+---
+
+A full-text search is used to perform natural-language searches on documents such as articles, websites, or other written formats.
```

Comment:
We want to add "New-in" to the top here? Or what is specifically new on this page?